setup.py

```diff
@@ -20,7 +20,7 @@
     long_description=readme,
     long_description_content_type="text/markdown",
     author="Helge Hecht, Maksym Skoryk",
-    author_email="helge.hecht@recetox.muni.cz, 245816@muni.cz",
+    author_email="helge.hecht@recetox.muni.cz, skoryk@mail.muni.cz",

```

Comment:
My `@mail.muni.cz` alias doesn't work anymore.
```suggestion
    author_email="helge.hecht@recetox.muni.cz, maksym.skoryk@recetox.muni.cz",

```